docker-compose.yml

```diff
@@ -94,6 +94,8 @@ services:
       - RSTUF_REDIS_SERVER="redis://redis"
       - SECRETS_RSTUF_TOKEN_KEY="secret"
       - SECRETS_RSTUF_ADMIN_PASSWORD="secret"
+      # If you want to test without authentication uncomment the next line:
```

Comment:
Hey just passing by - I believe we don't need this comment and just want a `- RSTUF_AUTH=true` line, which should be the default as described in the docs